Pages/quiz-rules.html

```diff
@@ -40,7 +40,10 @@ <h1 class="heading-sm">Rules</h1>
         I agree to above mentioned Rules.
       </label>
 
-      <a href="/pages/srk-quiz.html/" class="button btn-link"> Lets go!</a>
+
+      <a href="srk-quiz.html" class="button btn-link"> Lets go!</a>
+
+
```

Comment:
you could have formatted the code before pushing it

Reply:
Done, Thanks for bringing it to notice.